projects/hipblas-common/CMakeLists.txt

```
@@ -2,14 +2,17 @@
 # SPDX-License-Identifier: MIT
 
 cmake_minimum_required(VERSION 3.11)
-project(hipblas-common VERSION 1.2.0 LANGUAGES CXX)
+project(hipblas-common LANGUAGES CXX)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 include(dependencies)
 include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
 
+set(VERSION_STRING "1.3.0")
+rocm_setup_version(VERSION ${VERSION_STRING})
+
```

Will approve as is but is there a reason to avoid using the project function to set the version?

```suggestion
project(hipblas-common VERSION 1.3.0 LANGUAGES CXX)

list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")

include(dependencies)
include(GNUInstallDirs)
include(CMakePackageConfigHelpers)

rocm_setup_version(VERSION "${CMAKE_PROJECT_VERSION}")
```

---

That looks good to try in next PR